package.json

```diff
@@ -36,5 +40,8 @@
     "gulp-uglify": "~3.0.0",
     "gulp-zip": "~4.0.0",
     "jshint": "^2.9.5"
+  },
+  "dependencies": {
+    "ol": "^5.2.0"
```

Comment:
this will add a dependency to all users, not just those that want the ol5 sidebar

Reply:
You're totally right. Either I can remove it or move it in the `devDependencies`. I think the former would be better as sidebar focuses on other mapping libraries.